ufl/finiteelement/elementlist.py

```diff
@@ -200,6 +200,10 @@ def show_elements():
                   ("quadrilateral",))
 register_element2("BDMCF", 1, HDiv, "contravariant Piola", (1, None),
                   ("quadrilateral",))
+register_element2("SminusE", 1, HCurl, "covariant Piola", (1, None), cubes[1:3])
+register_element2("SminusF", 1, HDiv, "contravariant Piola", (1, None), cubes[1:2])
+register_element2("SminusDiv", 1, HDiv, "contravariant Piola", (1, None), cubes[1:3])
+register_element2("SminusCurl", 1, HCurl, "covariant Piola", (1, None), cubes[1:3])
```

Comment:
Why are these part of this change?

Reply:
I think these "Sminus" elements aren't in the FEniCS UFL yet? Anyways I have cleaned the commit history.